meson.build

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018 Endless Mobile, Inc.
+
+project('animations-dbus', 'c', version: '0.0.0', license: 'LGPL2.1+',
+    meson_version: '>= 0.42.0',
+    default_options: ['c_std=c99', 'warning_level=3'])
+
+gnome = import('gnome')
+pkg = import('pkgconfig')
+
+version_components = meson.project_version().split('.')
+api_version = version_components[0]
+api_name = '@0@-@1@'.format(meson.project_name(), api_version)
+libtool_version = '0.0.0'
+namespace_name = 'AnimationsDbus'
+
+gio = dependency('gio-2.0')
+gio_unix = dependency('gio-unix-2.0')
+glib = dependency('glib-2.0')
+gobject = dependency('gobject-2.0')
+
+config = configuration_data()
+config.set_quoted('ANIMATIONS_DBUS_VERSION', meson.project_version())
+configure_file(configuration: config, output: 'config.h')
+
+gdbus_targets = gnome.gdbus_codegen('animations-dbus-objects',
+    'data/com.endlessm.Libanimation.xml',
+    interface_prefix: 'com.endlessm.Libanimation',
+    namespace: 'AnimationsDbus')
+
+subdir('data')
+subdir('animations-dbus')
+
+requires = ['gio-2.0', 'gio-unix-2.0', 'glib-2.0', 'gobject-2.0']
```

Comment:
You can actually use the dependency objects declared above, so you don't repeat the names here.

Reply:
Handy, fixed.